src/components/KuadrantDNSPolicyCreatePage.tsx

```diff
@@ -248,7 +248,7 @@ const KuadrantDNSPolicyCreatePage: React.FC = () => {
                   </HelperText>
                 </FormHelperText>
               </FormGroup>
-              <NamespaceSelect selectedNamespace={selectedNamespace} onChange={setSelectedNamespace} />
+              <NamespaceSelect selectedNamespace={selectedNamespace} onChange={setSelectedNamespace}formDisabled={false} />
```

Comment:
this will be updated when update is in the other policies its just temp set to false